apps/ledger-live-desktop/src/renderer/hooks/useDeeplinking.js

```diff
@@ -43,10 +43,12 @@ export function useDeepLinkHandler() {
 
   const navigate = useCallback(
     (url: string, state?: any, search?: string) => {
-      if (url !== location.pathname) {
-        setTrackingSource("deeplink");
-        history.push({ pathname: url, state, search });
+      if (url === location.pathname && !search) {
```

Comment:
Can you add here a comment explaining this logic? (we need to check that we do not break any existing deeplink)